frontend/src/components/commons/LoginButton.jsx

```diff
@@ -8,16 +12,23 @@ const styles = {
 };
 
 class LoginButton extends Component {
+  static propTypes = {
+    // Props from Redux
+    setLoginState: PropTypes.func
+  };
+
   state = {
```

Comment:
Here you are storing the logged in state in the state and in redux. Instead you can delete the state, connect the props to the loggedIn value, and use props in the component instead of state.

```
<div>
        {!this.props.loggedIn && (
```

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/257/commits/a7bd265df32c5ee7fbdc24e7272cfe1d4e75c5c1